Gaming The Syllabus/UserSettings/EditorUserSettings.asset

```diff
@@ -0,0 +1,55 @@
+%YAML 1.1
```

Comment:
Really annoying these keep ending up in the repo... something for me to look into fixing in the future